WordPress/gutenberg
# edit-post: After closing welcome and content is empty, focus title field

## What?


Fires the useEffect to focus post title after welcome dialog closes.

## Why?


After closing welcome dialog, focus should be placed in a meaningful location.

## How?


useEffect now listens for the preferences store feature changes.

## Testing Instructions





1. Open a private browsing window and login.
2. Select Posts > Add New.
3. Close the welcome dialog.
4. Focus should now be placed in post title field.

## Screenshots or screencast
